scripts/import/import_all_sv_data.pl

```diff
@@ -235,7 +236,14 @@ =head1 CONTACT
   somatic_study_processing() if ($somatic_study == 1);
   structural_variation_set() if ($var_set_id);
   structural_variation_sample();
-  phenotype_feature();
+
+  # Fetch the default attrib id (only if there's phenotype data)
+  my $attrib_id;
+  if($phenotype_data) {
+    $attrib_id = get_attrib_id();
+  }
+  phenotype_feature($attrib_id);
```

Comment:
Why is phenotype_feature($attrib_id) not run only when there is $phenotype_data?

Reply:
It was just to avoid fetching the id when it's not going to be used.